src/api/deposit.js

```diff
@@ -48,8 +48,11 @@ module.exports = async (dvf, token, amount, starkPrivateKey) => {
   }
   // console.log({ data })
 
+  await dvf.contract.approve(token, dvf.token.toBaseUnitAmount(token, amount))
+
   const depositResponse = await post(url, { json: data })
-  const { status, transactionHash } = await dvf.contract.deposit(tempVaultId, token, amount)
+
+  const { status, transactionHash } = await dvf.contract.deposit(tempVaultId, token, amount); console.log({temp})
```

Comment:
console.log here should be removed

Reply:
it was hidden at the end of the line. took the opportunity to commit some style changes as well.